geometry_msgs/PointStamped[] array_objects_found
uint8 number_of_objects
uint8[] array_colors
